frontend/src/metabase/dashboard/components/Dashboard/Dashboard.styled.tsx

```diff
@@ -133,6 +133,18 @@ export const ParametersWidgetContainer = styled(FullWidthContainer)<{
           ? "var(--mb-color-border)"
           : getDashboardBodyBgColor(isNightMode)};
     `}
+
+    ${({ isNightMode }) =>
```

Comment:
Copy styled from https://github.com/metabase/metabase/blob/5a84cafd556af75c629899372dee16bea764db4d/frontend/src/metabase/public/components/EmbedFrame/EmbedFrame.module.css#L34-L68, but only carry over necessary style to make the filter looks like embedding dashboard.